resources/definitions/elegoo_base.def.json

```diff
@@ -0,0 +1,227 @@
+
+{
+  "version": 2,
+
+  "name": "ELEGOO Base Printer",
+
+  "inherits": "fdmprinter",   
+
+  "metadata": {
+      "author": "NARUTO",
+      "manufacturer": "ELEGOO",
+      "visible": false,
+      "file_formats": "text/x-gcode",
+      "platform": "elegoo_platform.3mf",
+      "platform_texture": "elegoobackplate.png",
+      "platform_offset": [-2.2, 14.5, -31],
+      "has_materials": true,
+      "has_variants": false,
+      "has_variant_materials": false,
+      "has_machine_materials": false,
+      "variants_name": "Print core",
+      "preferred_variant_name": "NEP 0.4",
+      "has_machine_quality": true,
+      "preferred_quality_type": "fine",
+      "preferred_material": "generic_pla_175",
+      "machine_extruder_trains":
+            {
+                "0": "elegoo_extruder_0"
+            },
+      "exclude_materials": [
+        "generic_nylon_175",
+        "generic_nylon",
+        "generic_hips_175",
+        "generic_hips",
+        "generic_gffcpe",
+        "generic_bvoh_175",
+        "generic_ccfpe",
+        "generic_cffcpe",
+        "generic_cpe_plus",
+        "generic_cpe_175",
+        "generic_cpe",
+        "generic_tpu",
+        "generic_pp",
+        "generic_pc",
+        "generic_pc_175",
+        "generic_pvc",
+        "generic_pva",
+        "generic_pva_175",
+        "generic_pvc_175",
+        "generic_pp"
+        ]
+  },
+  "settings":
+    {
+      "prime_tower_position_x_nep2d":
```

Comment:
Uh? THis isn't needed? There already is a primetower position. Why not just change that?

Reply:
Yes, I want the print head (Neptune2D machine) to move to the center of the PrimeTower when changing colors.

I'll delete it, but I really wish it could be set like this! Because I didn't find any problems running Cura5.0.